CLI/test/ressources/extractUsedCopies_EI/input/Copies.rdz.tcbl

```diff
@@ -5,9 +5,15 @@
        SOURCE-COMPUTER. IBM-370.
        DATA DIVISION.
        WORKING-STORAGE section.
-          01 PRTD20L. COPY  YPRTD20L.   
-          01 CONVEAL. COPY  YCONVEAL.
+          01 PRTD20L.  COPY  YPRTD20L.   
+          01 CONVEAL.  COPY  YCONVEAL.
           01 CONVEAL2. COPY  YCONVEAR.
+		  01 TRDF0L.   COPY  YTRDF0L.
+		  01 TRDF0LR.  COPY  YTRDF0LR.
+		  01 TRDF1L.   COPY  YTRDF1L.
+             COPY YTRDF2L  REPLACING ==:TRDF2L:== BY TDRF2L.
+             COPY YTRDF2LR REPLACING ==:TRDF2L:== BY TDRF2LR.
```

Comment:
It's not related to this issue, but there is a bug here.

` COPY YTRDF2LR REPLACING ==:TRDF2L:== BY TDRF2LR.` should produce a copy missing for YTRDF2LR.
But here, it's considered as a COPY managed as a suffixed COPY (CPY).
Our EI rule is:
Copy used with replacing, consider it as a standard Cobol copy, and don't apply suffixing. Also search the file named YTRDF2LR.cpy / .copy.
Copy without replacing and starts with "Y". If the COPY has 8 chars: apply suffixing and search for the file name with the first 7 chars of the COPY.
But this file doesn't exists.